### PR TITLE
Api key

### PR DESCRIPTION
**Pull request by adaext**
Move API key to local place and add a sample for `local.properties`.

### CODE REVIEW

**Review by mohammedkhider — CHANGES_REQUESTED** [review]
please replace "Ada" in the local.preperties.sample with "username" to avoid having your name in the code.

**Comment by adaext** [comment]
> please replace "Ada" in the local.preperties.sample with "username" to avoid having your name in the code.

Ok, I've done.

**Review by mohammedkhider — APPROVED** [review]

File: GNSSLogger/local.properties.sample [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file should *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+sdk.dir=C\:\\Users\\username\\AppData\\Local\\Android\\Sdk
+MAPS_API_KEY=Google Maps API key
```
Comment (mohammedkhider) [inline GNSSLogger/local.properties.sample:11]: Replace Google Maps API key with GoogleMapsApiKey

**Review by mohammedkhider — CHANGES_REQUESTED** [review]

File: GNSSLogger/local.properties.sample (hunk 1 above)
Comment (adaext) [reply GNSSLogger/local.properties.sample]: > Replace Google Maps API key with GoogleMapsApiKey

Ok, thanks.

**Pull request merged**